presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformUncorrelatedInPredicateSubqueryToSemiJoin.java

```diff
@@ -99,40 +99,29 @@ public InPredicateRewriter(PlanNodeIdAllocator idAllocator, SymbolAllocator symb
         @Override
         public PlanNode visitFilter(FilterNode node, RewriteContext<Void> context)
         {
-            InPredicateRewriteResult inPredicateRewriteResult = rewriteInPredicates(
+            PlanNode rewrittenNode = rewriteInPredicates(
                     context.defaultRewrite(node, context.get()),
                     ImmutableList.of(node.getPredicate()));
-            inPredicateMappings.add(inPredicateRewriteResult.inPredicateMapping);
 
             return new FilterNode(
-                    inPredicateRewriteResult.node.getId(),
-                    getOnlyElement(inPredicateRewriteResult.node.getSources()),
+                    rewrittenNode.getId(),
+                    getOnlyElement(rewrittenNode.getSources()),
                     replaceInPredicates(node.getPredicate()));
         }
 
-        private Expression replaceInPredicates(Expression expression)
```

Comment:
if possible please do not mix:
- moving functions around the class
- with other changes.